tests/integration/targets/ssh_key_info/tasks/main.yaml

```diff
@@ -39,11 +32,8 @@
           - by_label.ssh_key.created == create_ssh_key.body.created
 
   always:
-    - ignore_errors: yes
+    - ignore_errors: true
       block:
         - linode.cloud.api_request:
-            api_token: '{{ api_token }}'
```

Comment:
Unfortunately we'll have to keep these in for now since `ansible-test` does not natively pass-through shell environment variables.

Reply:
Actually, I did a bit more digging and we can at least simplify this by specifying `environment` at the bottom of a block:

```yaml
- name: my_tests
  block:
        ......
  always:
    - ignore_errors: yes
      block:
          .........

  environment:
    LINODE_UA_PREFIX: '{{ ua_prefix }}'
    LINODE_API_TOKEN: '{{ api_token }}'
```